File: projects/code_combat/8_Cloudrip_Mountain/363-Dance-Off/danceOff.js
// Move in sync with your dance partner to impress Pender Spellbane.

var partner = hero.findNearest(hero.findFriends());

var yDif = hero.pos.y - partner.pos.y;
var xDif = hero.pos.x - partner.pos.x;

while (true) {
    hero.move({"x": partner.pos.x + xDif, "y": partner.pos.y + yDif});
}

File: projects/code_combat/8_Cloudrip_Mountain/363-Dance-Off/dance_off.py
partner = hero.findNearest(hero.findFriends())
yDif = hero.pos.y - partner.pos.y
xDif = hero.pos.x - partner.pos.x

while True:
    hero.move({"x": partner.pos.x + xDif, "y": partner.pos.y + yDif})
